### ISSUE TITLE
Do not warn about externals when using transform runtime.

### ISSUE THREAD

**Issue by swernerx** [rollup/rollup-plugin-babel#105]
For bundling our library we are using "transform-runtime" together with `polyfill: false` flag. Even if setting `runtimeHelpers: true` as suggested in the documentation we are seeing this confusing message: 

```
The '${helper}' Babel helper is used more than once in your code. 
```

which should probably not happen in my opinion. Looking through the code it seems changing this in https://github.com/rollup/rollup-plugin-babel/blob/master/src/index.js#L64 should be enough:

```js
} else {
```

into 

```js
} else (helpers !== RUNTIME) {
```

**Comment by swernerx** [comment]
BTW: We have one bundle were we do not transpile es2015 classes (for modern clients/nodejs). In this case the whole detection in `preflightCheck.js` might be broken as it explicitely check for usage of the class helper to detect the runtime usage.

**Comment by Rich-Harris** [comment]
Fixed in 2.7.1 – thanks
